staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/status_strategy_test.go

```diff
@@ -136,3 +139,182 @@ func TestPrepareForUpdate(t *testing.T) {
 		}
 	}
 }
+
+func strPtr(s string) *string { return &s }
+
+func TestStatusStrategyValidateUpdate(t *testing.T) {
+	strategy := statusStrategy{}
+	kind := schema.GroupVersionKind{
+		Version: "v1",
+		Kind:    "foo",
+		Group:   "test",
+	}
+	strategy.customResourceStrategy.validator.kind = kind
+	v := "v1"
+	structuralSchemas := map[string]*structuralschema.Structural{}
+	structuralSchema, _ := structuralschema.NewStructural(&apiextensions.JSONSchemaProps{
+		Type: "object",
+		AdditionalProperties: &apiextensions.JSONSchemaPropsOrBool{
+			Allows: true,
+		},
+	})
+	structuralSchema1, _ := structuralschema.NewStructural(&apiextensions.JSONSchemaProps{
+		Type:       "object",
+		XListType:  strPtr("set"),
+		Properties: map[string]apiextensions.JSONSchemaProps{},
+	})
+	structuralSchema.AdditionalProperties.Structural = structuralSchema1
+	structuralSchemas[v] = structuralSchema
+	strategy.structuralSchemas = structuralSchemas
+
+	ctx := context.TODO()
+	tcs := []struct {
+		name    string
+		old     *unstructured.Unstructured
+		obj     *unstructured.Unstructured
+		isValid bool
+	}{
+		{
+			name: "bothValid",
+			// changes to spec are ignored
```

Comment:
this is a confusing comment to make in a validate test... ignoring spec changes happens in PrepareForUpdate

Reply:
Removed.